### PR TITLE
User dashboard

### PR DESCRIPTION
**Pull request by sebastianottow**
Added Header and NavBar.
Header included Logo with LINK to Home and loads a profile image.
profile image has no function yet, since profile page still to be developed.
NavBar includes the "Home" button and links to Home.
Add Car Form added in - further functionality following in UserStories.

### CODE REVIEW

File: src/components/Header/Header.styled.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components';
+
+const StyledHeader = styled.div`
```
Comment (irenepapaspyratos) [inline src/components/Header/Header.styled.js]: Wouldn't \<header> be better than just \<div> ? 🤔

File: src/components/Header/Header.styled.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+		background-color: grey;
+		margin-right: 2rem;
+		display: flex;
+		cursor: pointer;
+		position: relative;
+		object-fit: cover;
+		overflow: hidden;
+	}
+`;
+
```
Comment (irenepapaspyratos) [inline src/components/Header/Header.styled.js:20]: Nested tags are not appreciated. Maybe have a look at the related codeSandboxes in our channel?

File: src/components/NavBar/NavBar.styled.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	bottom: 0;
+	left: 0;
+	z-index: 1000;
+
+	a {
+		align-self: center;
+		color: hsl(264, 100%, 50%);
+	}
+`;
+
```
Comment (irenepapaspyratos) [inline src/components/NavBar/NavBar.styled.js:17]: Nested tags issue.

File: src/components/Layout.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import NavBar from './NavBar/NavBar';
+
+export default function Layout({ children }) {
+	// styles the main html tag
+	const styles = {
+		margin: '9vh 0 0 0 ',
+		display: 'flex',
+		flexDirection: 'column',
+	};
+
```
Comment (irenepapaspyratos) [inline src/components/Layout.js]: As your styles-variable is fixed without any variants, you could build a Main.styled.js with those attributes to keep this component clean.

File: pages/index.js [hunk 5]
```diff
@@ -7,7 +7,6 @@ export default function Home() {
 
 	return (
 		<>
-			<h1>caars</h1>
 			<AddButton onClick={() => setShow(true)}>+</AddButton>
```
Comment (irenepapaspyratos) [inline pages/index.js]: The Button-component should rather export one single Button with variants than different Button-types by themselves.

**Comment by irenepapaspyratos** [comment]
Hi Sebastian, I think, regarding the last workshop, you need to reorganize your styled components a tiny bit. Besides that, it looks fine to me. 🙃

**Review by irenepapaspyratos — APPROVED** [review]

File: src/components/Header/Header.styled.js (hunk 1 above)
Comment (sebastianottow) [reply src/components/Header/Header.styled.js]: yes sure it. adjusted accordingly :)

File: pages/index.js (hunk 5 above)
Comment (sebastianottow) [reply pages/index.js]: adjusted and done.

File: src/components/Layout.js (hunk 4 above)
Comment (sebastianottow) [reply src/components/Layout.js]: Done

File: src/components/Header/Header.styled.js (hunk 2 above)
Comment (sebastianottow) [reply src/components/Header/Header.styled.js:20]: Done accordingly to comment

File: src/components/NavBar/NavBar.styled.js (hunk 3 above)
Comment (sebastianottow) [reply src/components/NavBar/NavBar.styled.js:17]: Done accordingly to comment

**Pull request merged**